Applications/WebPortal/WebApp/library/source/pages/account_directory_page/account_entry.ts

```diff
@@ -0,0 +1,28 @@
+import * as Beam from 'beam';
+import * as Nexus from 'nexus';
+
+/** A object that represents a account entry and its roles. */
```

Comment:
A class is not an object. You can remove "An object that" altogether and just say "Represents an account entry and its roles."